packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1454,7 +1455,30 @@ async function handler(
       metadata: reqBody.metadata,
       attendees: {
         createMany: {
-          data: attendeesData,
+          data: [
+            ...evt.attendees.map((attendee) => {
+              //if attendee is team member, it should fetch their locale not booker's locale
+              //perhaps make email fetch request to see if his locale is stored, else
+              const retObj = {
+                name: attendee.name,
+                email: attendee.email,
+                timeZone: attendee.timeZone,
+                locale: attendee.language.locale,
+              };
+              return retObj;
+            }),
+            // Have this for now until we change the relationship between bookings & team members
+            ...(evt.team?.members
+              ? evt.team.members.map((member) => {
+                  return {
+                    email: member.email,
+                    name: member.name,
+                    timeZone: member.timeZone,
+                    locale: member.language.locale,
+                  };
+                })
+              : []),
```

Comment:
Do we want ALL team members as attendees? Does this takes into account round-robin?

Reply:
The flow isn't nice, but it goes from users -> availableUsers -> teamMemberPromises -> teamMembers -> team.members = teamMembers.